asteroid/losses/sinkpit_wrapper.py

```diff
@@ -0,0 +1,192 @@
+import torch
+from torch import nn
+import pytorch_lightning as pl
+
+from . import PITLossWrapper
+
+
+class SinkPITLossWrapper(nn.Module):
+    r"""Permutation invariant loss wrapper.
+
+    Args:
+        loss_func: function with signature (targets, est_targets, **kwargs).
+        n_iter (int): number of the Sinkhorn iteration (default = 200).
+            Supposed to be an even number.
+        hungarian_validation (boolean) : Whether to use the Hungarian algorithm
+            for the validation. (default = True)
+
+        `loss_func` computes pairwise
+        losses and returns a torch.Tensor of shape
+        :math:`(batch, n\_src, n\_src)`. Each element
+        :math:`[batch, i, j]` corresponds to the loss between
+        :math:`targets[:, i]` and :math:`est\_targets[:, j]`
+        It evaluates an approximate value of the PIT loss
+        using Sinkhorn's iterative algorithm.
+        See :meth:`~PITLossWrapper.best_softperm_sinkhorn`
+        and http://arxiv.org/abs/2010.11871
+
+    Examples
+        >>> import torch
```

Comment:
This is not something that we have to fix in this PR, but still: Since none of these lines actually show the return values of any of the expressions, I think we can do away with all of the `>>>`. (Btw they aren't "correct" since in the multiline expressions below the lines should start with `...`.) What do you think @mpariente?

Reply:
You mean remove them right? 
I don't know how that renders in the docs without `>>>`, I'd have to check.

Reply:
This is Sphinx, right?

You can either do

```
Examples::

	from ... import ...
```

which renders as `Examples: ...`

or

```
Examples ::

	from ... import ...
```

which renders as `Examples ...` (no `:`)

Reply:
I don't know sphinx that well, it's always a mistery why some things work and others don't.

Reply:
If you have a pointer to a succinct and good guide to Sphinx, I'll take it. 
I have a unfinished document called `writing_good_docstrings`, I still don't know how to :roll_eyes:

Reply:
Haha yeah I was thinking about that too. I also don't have a clue. Maybe sometime we can spend some time researching this and coming up with a guide. Sometime...

Reply:
Maybe just look at PyTorch or NumPy docstrings